hw5.rb

```diff
@@ -0,0 +1,19 @@
+puts  "#2" 
+names.each { |el| print el.length };
+puts
+puts  "#3"
+names.each { |el| print el. downcase }
+puts
+puts  "#4"
+puts h=Hash[names.map{|el| [el,el.size]}]
+puts
+puts  "#5"
+puts  ab=h.invert.max&.last
+puts 
+puts  "#6"
+puts names.index(ab)
+puts 
+puts "#7"
+puts names.sort_by(&:length)
```

Comment:
Print an array which begins with the longest name from `names` and ends with the last element of `names`